### ISSUE TITLE
Bat.to

### ISSUE THREAD

**Issue by YahyeDeero** [tachiyomiorg/tachiyomi-extensions#17010]
### Source information

Bat.to 1.4.31

### Source language

English 

### Steps to reproduce

1. Open
2. Wait

### Expected behavior

It should have work

### Actual behavior

java.net.connectException: Failed to connect to/

### Tachiyomi version

0.14.6

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by ChikaTempest** [comment]
Source information
Bato.to 1.4.31

Source language
Multi

Source new URL
https://dto.to/

Other details
No response

Acknowledgements
 I have searched the existing issues and this is a new ticket, NOT a duplicate or related to another open or closed issue.
 I have written a short but informative title.
 I have updated all installed extensions.
 I have opened WebView and checked that the source URL is not updated yet.
 I will fill out all of the requested information in this form.
